### DISCUSSION TITLE
A Zip File (5.4 MiB) of the PDFs for The EX-T3-WiFi Documentation

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#72] [General]
I created a ZIP file of PDF files for all the documentation and uploaded them here!

I like to keep a hard-copy book of the reference material for my visitors to refer to and have them enjoy the  pretty pictures that Alex created.

When an updated MD file occurs, I will try to update the ZIP.

Dave Clifford

[EX-T3-Wifi-Docs.zip](https://github.com/DCC-EX/EX-T3-WiFi/files/10889695/EX-T3-Wifi-Docs.zip)

**Comment by Hargard339** [comment]
Hi Dave
Thank you for all your effort. Have mine all built ready for testing. 
Thanks again.
Antony.
